keyboards/takashicompany/baumkuchen/keymaps/via/config.h

```diff
@@ -0,0 +1,6 @@
+// Copyright 2023 takashicompany (@takashicompany)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define DYNAMIC_KEYMAP_LAYER_COUNT 29
```

Comment:
Why 29?

Reply:
@drashna 
This is because I checked the number of layers that can be registered with VIA on this keyboard and it was 29 layers.